scripts/tippecanoe/1.22.2/.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: generic
+
+sudo: false
```

Comment:
Thanks @springmeyer. I'm not sure which package I copied the previous tippecanoe version's `.travis.yml` from, since it doesn't seem to quite match any of the others.

I don't think I have the permissions to run `mason trigger` so I would appreciate your help.